org.eclipse.jdt.ui.tests/ui/org/eclipse/jdt/ui/tests/jarexport/FatJarExportTests.java

```diff
@@ -1,1017 +0,0 @@
-/*******************************************************************************
```

Comment:
You should have deleted your change only, not the whole file.

Reply:
@carstenartur If you put back the file with whatever changes that are needed for JUnit5, I can merge.

Reply:
> @carstenartur If you put back the file with whatever changes that are needed for JUnit5, I can merge.

Sure, I hope I can do it at the weekend. When I'm short on mobile data this month could be the 3rd of October. I did not find a way this can be repaired using the github web interface. In this specific regard gerrit was better I fear. I have removed file changes a number of times using the web interface of gerrit in the past (this functionality unfortunatly was behind a name like "delete file" in gerrit otoh what does something completely different in github)

Reply:
Fixed